web-services/model/src/test/java/datawave/webservice/query/model/ModelBeanTest.java

```diff
@@ -91,8 +91,24 @@ public void setup() throws Exception {
         
         URL m1Url = ModelBeanTest.class.getResource("/ModelBeanTest_m1.xml");
         URL m2Url = ModelBeanTest.class.getResource("/ModelBeanTest_m2.xml");
+
+         /*
+         * In jdk 9+, java.lang.Package.getPackageInfo() may fail to load a package-info.class,
+         * which is needed to resolve our JAXB package-level annotations. Here, this results in:
+         *
+         * "javax.xml.bind.UnmarshalException: unexpected element (uri:"http://webservice.datawave/v1", local:"Model").
+         *    Expected elements are <{}Model>"
+         *
+         * As a workaround, we can force it to load here in the same manner as in jdk 8's Package.getPackageInfo(),
+         * though there's likely a "better" (i.e., module-focused) way of handling this in jdk 9+
+         *
+         * E.g., see https://stackoverflow.com/questions/52157040/
+         */
+        Class.forName("datawave.webservice.model.package-info", false, this.getClass().getClassLoader());
```

Comment:
I note that other classes have the same JAXB setup but don't have this problem, such as [here](https://github.com/NationalSecurityAgency/datawave/blob/integration/warehouse/query-core/src/test/java/datawave/query/cardinality/TestCardinalityConfiguration.java#L31) and [here](https://github.com/NationalSecurityAgency/datawave/blob/integration/web-services/client/src/test/java/datawave/webservice/query/metric/QueryMetricTest.java#L149).

I also note that PowerMockRunner seems to be the main contributing factor here, given that when `@RunWith(PowerMockRunner.class)` is commented out, the JAXB deserialization works fine as-is, no workaround required